test/Microsoft.OpenApi.Tests/Services/OpenApiCSDLConversionTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+using Microsoft.OpenApi.Hidi;
+using Microsoft.OpenApi.Services;
+using Xunit;
+
+namespace Microsoft.OpenApi.Tests.Services
+{
+    public class OpenApiCSDLConversionTests
+    {
+        [Fact]
+        public void ReturnConvertedCSDLFile()
+        {
+            // Arrange
+            var filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "UtilityFiles\\Todo.xml");
+            var fileInput = new FileInfo(filePath);
+            var csdlStream = fileInput.OpenRead();
+
+            // Act
+            var openApiDoc = OpenApiService.ConvertCsdlToOpenApi(csdlStream);
```

Comment:
is there a test checking ProcessOpenApiDocument elsewhere?

Reply:
No there isn't